tests/test_api_validate.py

```diff
@@ -6137,14 +6137,14 @@ def test_02_autenticate(self):
                  'username': 'cornelius',
                  'refilltoken': '79906cc20567c6ca1e4b452500bb0662e107ce0fa14742c95b7e5c6a1417a519f829318622c1d569',
                  'repsonse': {
-                     'pubkey': 'a50102032620012158203f98500ddcedc3aa16d34ae9b7c12...ea3e37193f8f3d', 
-                     'credential_id': 'RuBlEInU7ycsILST7u6AoT7rdqN...4xM2SHltwbtBwApFYnbQXO8g5bgrb4kFh1NErnzsT6xA',
-                     'rpId': 'netknights.it'}, 
```

Comment:
You are changing the API here. 
This might break installations in the field, that update the privacyIDEA server but do not update clients.
I see no necessity to do this.

Also, please ensure to follow our development guideline:
https://github.com/privacyidea/privacyidea/blob/master/CONTRIBUTING.md#develop
I.e. please create an issue before adding/changing code.

Reply:
This has not been released yet, so it does not break anything. It changes the names so that it uses the same convention as the already released API which uses camelcase for webauthn things:
 
![grafik](https://github.com/privacyidea/privacyidea/assets/29949516/3745fe03-9bfa-4e42-8ae1-aec1d02acde0)